src/xdocs/config_whitespace.xml

```diff
@@ -16,6 +16,266 @@
       </macro>
     </section>
 
+    <section name="ChainedMethodCallWrap">
+      <p>Since Checkstyle 10.2</p>
+      <subsection name="Description" id="ChainedMethodCallWrap_Description">
+        <p>
+          Checks whether method calls specified by identifier pattern are wrapped.
+          Line wrapping means to insert line breaks in code to make it more readable and
+          adhere to line length standards.
+        </p>
+        <p>
+          Rationale: Long chained method calls are difficult to read and comprehend and sometimes
+          require horizontal scrolling to read the complete method call chain. Wrapping them onto
+          the next line/s enhances readability.
```

Comment:
I am not sure if all users will understand this usage of optional plural. I would recommend having explicit "line or lines", or at least do more common form of `line(s)`.  Further suggestion: `Wrapping method calls onto the following line or lines enhances readability.`

Reply:
Done, I went on with the line(s) approach, I have seen it being used in many places, it isn't hard to comprehend.

Reply:
Resolved.